extensions/data-transfer/portability-data-transfer-smugmug/src/main/java/org/datatransferproject/transfer/smugmug/photos/SmugMugInterface.java

```diff
@@ -110,11 +108,14 @@ SmugMugAlbumsResponse getAlbums(String url) throws IOException {
   SmugMugAlbumResponse createAlbum(String albumName) throws IOException {
     // Set up album
     Map<String, String> json = new HashMap<>();
+    String niceName = "Copy-of-" + albumName.replace(' ', '-');
```

Comment:
Should we just change it so it strips non a-z chars?

Reply:
I don't want to do that until we talk to the smugmug team. I'm worried about other alphabets, and name overlaps.

For now this is just a reversion, I'll take another shot at making it work in the future.